src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2346,5 +2346,10 @@ internal enum ErrorCode
         // Note: you will need to do the following after adding warnings:
         //  1) Re-generate compiler code (eng\generate-compiler-code.cmd).
         //  2) Update ErrorFacts.IsBuildOnlyDiagnostic (src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs)
+
+        // PROTOTYPE: Condense
```

Comment:
I'll address these prototypes after we do the test plan review with the compiler team. They're here for now to make it easier for me to merge in `main`.